pymedphys/_experimental/pseudonymisation/__init__.py

```diff
@@ -20,10 +20,11 @@
 from pymedphys._dicom.anonymise import (
     anonymise_directory,
     anonymise_file,
+    get_baseline_keyword_vr_dict,
     get_default_identifying_keywords,
 )
 
-from . import strategy
+from . import strategy  # pylint: disable=import-self
```

Comment:
I've never seen this pylint error before. Would this be able to be overcome by defining `is_valid_strategy_for_keywords` in a separate file?

Reply:
> I've never seen this pylint error before. Would this be able to be overcome by defining `is_valid_strategy_for_keywords` in a separate file?

it wasn't that function in particular, the same import is used in anonymise, presumably for accessing the default keywords or at least the default strategy for anonymisation.

Reply:
I managed to work it out :). Details over at https://github.com/pymedphys/pymedphys/pull/1044#issuecomment-690034690